repository-data/application/src/main/resources/hcm-content/content/documents/administration/admin-command-interface.yaml

```diff
@@ -0,0 +1,43 @@
+---
+/content/documents/administration/admin-command-interface:
+  /admin-command-interface[1]:
+    common:entries:
+      type: string
+      value: []
+    common:searchable: true
+    hippo:availability: []
+    hippostd:retainable: false
+    hippostd:state: draft
+    hippostdpubwf:createdBy: admin
+    hippostdpubwf:creationDate: 2022-01-18T20:13:48.231Z
+    hippostdpubwf:lastModificationDate: 2022-01-18T20:13:48.231Z
+    hippostdpubwf:lastModifiedBy: admin
+    hippotranslation:id: cc919a10-e73d-4370-a8ec-88c6b2d40ff9
+    hippotranslation:locale: document-type-locale
+    jcr:mixinTypes:
+    - mix:referenceable
+    jcr:primaryType: common:stringlist
+  /admin-command-interface[2]:
```

Comment:
@ziemsky I would keep this one in mind in case something goes wrong. I'm unsure if get(0) will always be safe here.

Reply:
Will do. Thanks for the warning, @Vlets :+1: 

So far, in my tests, there was always exactly one sub-node returned as long as the document has a published version. If the document is not published, the '`get(0)` invoking method will not even be called. Should the invocation ever be done in a different context, an error will be logged with a full stack trace which, hopefully, should be enough for a developer to understand what went wrong on that rare occasion when this feature is at all used.

Having said, I'll have a think and will consider implementing some cleaner way of dealing with this potential situation.

Reply:
Fixed: `AdminCommandProvider` is now more resilient to situations where the structure of the Admin Command Interface document is not _exactly_ as it requires.